### DISCUSSION TITLE
Support binding search functions to Ctrl+Space using bind -k

### DISCUSSION THREAD

**Discussion by atsarevskiy** [PatrickF1/fzf.fish#279] [Ideas]
Curently fzf_configure_bindings doesn't support "bind -k" format which doesn't allow to configure ctrl+space binding as it is only can be set using -k syntax see https://github.com/fish-shell/fish-shell/issues/3189#issuecomment-531523446

**Comment by PatrickF1** [comment]
Hi Daxzel, is the specific issue that you want to use Ctrl+Space to activate a fzf.fish search command but that isn't currently supported?

**Reply by PatrickF1** [reply]
Oh I see, so Ctrl+space generates the null character, and is the only key binding that needs to be binded using `bind -K`, not by normal bind. Did I understand your problem correctly?

**Reply by atsarevskiy** [reply]
Yes, that exactly correct. While I solved my problem by looking into the script and setting a bind manually it still could be confusing for others.

**Reply by PatrickF1** [reply]
Gotcha. It'd be a sizable refactor to update it for not much additional functionality/flexibility, in my eyes. However, I will update the troubleshooting to call it out. Thanks!
